test/e2e/app_suffix_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/k14s/kapp/pkg/kapp/app"
+)
+
+var (
+	yaml1 string = `
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-primary
+spec:
+  ports:
+  - port: 6380
+    targetPort: 6380
+  selector:
+    app: redis
+    tier: backend
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: redis-config
+data:
+  key: value
+`
+
+	yaml2 string = `
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: redis-config
+data:
+  key: value2
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: redis-config2
+data:
+  key: value
+`
+)
+
+func TestAppSuffix_CreateUpdateDelete(t *testing.T) {
```

Comment:
We don't have a hard and fast rule against using `undescores` but I think we follow PascalCase in kapp.
Also, I am thinking if we should use the existing CreateUpdateDelete test.

Reply:
cool, funnily enough I did have it there. But it felt like it was testing two distinct things. 1 that tests that a deploy works as expected (resources created) and this one which specifically checks the config-map exists and has the suffix